toast-er.html

```diff
@@ -308,22 +308,26 @@
 
         // Get configuration of the toast.
         var bread = this._breads[0],
-          duration = bread.duration || this.duration,
-          type = bread.type || this.type,
+          buttonEvent = bread.buttonEvent || this.buttonEvent,
+          buttonText = bread.buttonText || this.buttonText,
+          hasButton = buttonText.length != 0,
+          duration = bread.duration || (hasButton ? 10000 : this.duration),
```

Comment:
@wincinderith Yes, it's the purpose. If the toaster has a button, it should be displayed at least 10 sec. I don't know if human can read carefully a message and take the choice to click or not in less time. Maybe 5 sec is enough. What do you think about this "feature" ?